exercises/option/option3.rs

```diff
@@ -0,0 +1,20 @@
+// option3.rs
+// Make me compile! Execute `rustlings hint option3` for hints
+
+// I AM NOT DONE
+
+struct Point {
+    x: i32,
+    y: i32,
+}
+
+fn main() {
+    let y: Option<Point> = Some(Point { x: 100, y: 200 });
+
+    match y {
+        Some(p) => println!("Co-ordinates are{},{} ", p.x, p.y),
+        _ => println!("no match"),
+    }
+    // Fix without deleting this line.
```

Comment:
you can move this comment after line 19 so it's more obvious which line "this line" is referring to

Reply:
Moved the comment to be inline which I think makes the intent clear.

Reply:
oh yeah sorry that's what I meant